lib/common/client/base.rb

```diff
@@ -32,17 +30,7 @@ def connection
           connection = config.connection
           handlers = connection.builder.handlers
 
-          if handlers.include?(Faraday::Adapter::HTTPClient) &&
```

Comment:
Lets avoid making these types of changes to client/base unless truly warranted. Everything application specific should be handled by overridden version of client/base or configuration/base that sub classes can inherit from.

Pretend these are gems that are not within your control to modify.

Reply:
I didn't change any functionality; I extracted it to a module so it can also be used by `EVSS::BaseService` since it has no Faraday-related checks currently.  Should it not?  
Should we not add to these checks as we identify ordering imperatives?

Reply:
I'm not sure really, I guess we could always go in and clean things up a bit later. But if HTTPClient isn't used very frequently maybe it's just a gotcha that devs live with? Or Maybe we create a new client that inherits from Base that overrides that method that's specifically suited to HTTPClient? Use your discretion, but I worry about 2 things here:

1. Are we introducing unnecessary cruft that only affects 10% of all services/clients? If so it should live elsewhere.

2. Are we introducing "magic" that could defy the ["principle of least astonishment"](https://en.wikipedia.org/wiki/Principle_of_least_astonishment) in such a way that it ends up producing difficult to debug scenarios for future developers? Are we simplifying logic or adding complexity? And if adding complexity, is the complexity worth it?